package.json

```diff
@@ -185,7 +185,8 @@
         "posthtml-transform": "^1.0.6",
         "prettier": "^1.16.4",
         "pretty-quick": "^1.10.0",
-        "prismjs": "^1.15.0",
+        "prism-themes": "^1.3.0",
+        "prismjs": "^1.17.1",
```

Comment:
Need to remove all of these Prism related changes. Was attempting to get something more accessible in the light range, but none of their standard themes pass AA a11y standards even.